apps/judicial-system/api/src/app/modules/case/dto/createCase.input.ts

```diff
@@ -15,8 +16,8 @@ export class CreateCaseInput implements CreateCase {
   readonly type!: CaseType
 
   @Allow()
-  @Field(() => String, { nullable: true })
-  readonly indictmentSubType?: IndictmentSubType
+  @Field(() => GraphQLJSONObject, { nullable: true })
+  readonly indictmentSubTypes?: IndictmentSubtypeMap
```

Comment:
Please rename this to indictmentSubtypes (not ...SubTypes)

Reply:
Actually subtype and sub-type are both valid, but subtype is more common. Will change.